BlockchainSdk/Common/Blockchain.swift

```diff
@@ -444,7 +444,7 @@ extension Blockchain {
             } else {
                 return [
                     URL(string: "https://eth.nownodes.io/\(nowNodesApiKey)")!,
-                    URL(string: "https://eth.getblock.io/mainnet?api_key=\(getBlockApiKey)")!,
+                    makeGetBlockJsonRpcProvider(for: .ethereum(testnet: false)),
```

Comment:
может лучше через `self`

```suggestion
                    makeGetBlockJsonRpcProvider(for: self),
```